react-validated-input/my-app/src/ValidatedInput.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react';
+import { FaCheck, FaTimes } from 'react-icons/fa';
+
+export function ValidatedInput() {
+  const [value, setValue] = useState('');
+  let message = '';
+  if (value === '') {
+    message = 'A password is required.';
+  }
+  if (value.length < 8) message = 'Your password is too short. ';
```

Comment:
This needs to be in an `else-if`. Otherwise this will execute on an empty value because length 0 is less than 8.